### PR TITLE
Update hardware.json - change locationSet for "Würth"

### PR DESCRIPTION
**Pull request by phd0**

### CODE REVIEW

File: data/brands/office/insurance.json [hunk 1]
```diff
@@ -856,17 +856,7 @@
...
-          "gr",
-          "hu",
-          "it",
-          "ro",
-          "sk",
-          "tr"
-        ]
-      },
+        "include": [ "bg", "cn", "fr", "gr", "hu", "it", "ro", "sk", "tr"]},
       "matchTags": ["amenity/bank|groupama"],
```
Comment (arch0345) [inline data/brands/office/insurance.json:860]: `matchTags` are used to match existing features with different tags to this entry. If Groupma locations tagged with `amenity=bank` should be changed to `office=insurance`, the suggestion below should be committed to use correct formatting (see [this wiki page](https://github.com/osmlab/name-suggestion-index/wiki/Contributing#matchnamesmatchtags-optional))

```suggestion
       "matchTags": ["amenity/bank"],
```

File: data/brands/shop/hardware.json [hunk 2]
```diff
@@ -309,19 +309,7 @@
...
-          "de",
-          "fi",
-          "fr",
-          "it",
-          "no",
-          "sk"
-        ]
-      },
-      "note": "https://github.com/osmlab/name-suggestion-index/issues/5500",
+      "locationSet": {"include": ['001'], "exclude": ['us', 'ca', 'jp', 'ru', 'tw', 'cn']},
```
Comment (arch0345) [inline data/brands/shop/hardware.json:312]: Strings in json require double quotes

```suggestion
      "locationSet": {"include": ["001"], "exclude": ["us", "ca", "jp", "ru", "tw", "cn"']},
```

**Review by arch0345 — CHANGES_REQUESTED** [review]

File: data/brands/shop/hardware.json (hunk 2 above)
Comment (phd0) [reply data/brands/shop/hardware.json:312]: yes, my tests on https://location-conflation.com/?referrer=nsi misled me, it went well with simple quillemets

**Review by arch0345 — CHANGES_REQUESTED** [review]
Each entry needs to include the primary tag specified in the file name (ie `amenity=bank`, `office=insurance`). Since the validator matches existing features with the primary tag, there's no need for `matchTags` in this scenario.

**Pull request merged**